src/pages/HomePage/ActionButtons.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import {
+ Row, Col, Icon, notification,
+} from 'antd';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import PropTypes from 'prop-types';
+
+import { columnSeparator, actionIcon, deleteButton } from './styles';
+import DeleteAppModal from './DeleteAppModal';
+import { SCALR_URL, IMPORTER_LINK } from '../../constants/config';
+import {
+	getCredentialsFromPermissions,
+	getReadCredentialsFromPermissions,
+} from '../../batteries/utils';
+
+class ActionButtons extends React.Component {
+	state = {
+		deleteModal: false,
+	};
+
+	handleDeleteModal = () => {
+		const { deleteModal: currentValue } = this.state;
+		this.setState({
+			deleteModal: !currentValue,
+		});
+	};
+
+	handleClone = (e) => {
+		const { appName, appId, permissions } = this.props;
+		const { username, password } = permissions.results[0];
+		const parameters = {
+			platform: 'appbase',
+			importFrom: {
+				appname: appName,
+				hosturl: `https://${username}:${password}@${SCALR_URL}`,
+			},
+		};
+		window.open(`${IMPORTER_LINK}${JSON.stringify(parameters)}`, '_blank');
+		e.stopPropagation();
+	};
+
+	copyWriteKey = () => {
```

Comment:
Why do we not have `e.stopPropagation()`s in other action-btns events handlers? 🤔